src/components/BGImage/BGImage.styles.js

```diff
@@ -28,7 +28,7 @@ export const Content = styled.div`
   height: 100%;
   width: 100%;
   z-index: 2;
-  @media (max-width: 768px){
-    top: 40%;
+  @media (max-width: 992px){
+    top: 35%;
```

Comment:
The fix is not simply updating the `top` value. It requires drilling through the CSS of the `BGImage` component and updating the `position` value to `relative` and `absolute` based on resolution.